bindings/matrix-sdk-crypto-js/package.json

```diff
@@ -37,12 +37,9 @@
         "node": ">= 10"
     },
     "scripts": {
-        "build": "cross-env RUSTFLAGS='-C opt-level=z' WASM_BINDGEN_WEAKREF=1 wasm-pack build --release --target nodejs --scope matrix-org --out-dir ./pkg",
+        "build": "./scripts/build.sh",
         "test": "jest --verbose",
         "doc": "typedoc --tsconfig .",
-        "prepack": "npm run build && npm run test",
-        "pack": "wasm-pack pack",
-        "prepublish": "npm run pack",
-        "publish": "wasm-pack publish"
```

Comment:
I've removed these for now as I don't think they'll do the right thing, and they seemed to be a footgun.